src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -7705,11 +7703,30 @@ int P1
         }
 
         [Fact]
-        public void TestDeprecatedAttributeTH1()
+        public void TestMultipleDeprecatedAttributes()
         {
-            var source1 = @"
-using System;
-using Windows.Foundation.Metadata;
+            var source =
+@"using Windows.Foundation.Metadata;
+class C
+{
+    [Deprecated(""Removed"", DeprecationType.Remove, 0)]
+    [Deprecated(""Deprecated"", DeprecationType.Deprecate, 0)]
+    static void F() { }
+    static void Main()
+    {
+        F();
+    }
+}";
+            var compilation = CreateCompilation(source, WinRtRefs, TestOptions.ReleaseDll);
+            // Should report 'error' rather than 'warning'. See https://github.com/dotnet/roslyn/issues/18755.
```

Comment:
>// Should report 'error' rather than 'warning'. See https://github.com/dotnet/roslyn/issues/18755. [](start = 12, length = 98)

I believe the current behavior is by design.